### PR TITLE
move plugins to one folder

### LINKED ISSUE #1348: Determine how plugins are used by pluginType.

### PR DESCRIPTION
**Pull request by soofstad**
## What does this pull request change?
- Moved 'dmt/common' out of 'plugins' and renamed it to 'dmt-core'.
- Moved all plugins into on folder called 'plugins'. Removed "app" and "ui" folder.

## Why is this pull request needed?
There is no distinction between 'app plugins' and 'ui plugins' for DMT when loading apps.
There are only packages that exports multiple plugins, of either 'page'
or 'ui' type. The file structure should reflect this.

## Issues related to this change
closes #1348

### CODE REVIEW

File: web/packages/plugins/simpos-ui/src/simpos/views/report.js [hunk 1]
```diff
@@ -22,7 +22,7 @@
 
 import styled from 'styled-components'
 import { useTable } from 'react-table'
-import Plot from 'react-plotly.js'
+import Plot from '../../../../../../node_modules/react-plotly.js/react-plotly'
```
Comment (netr0m) [inline web/packages/plugins/simpos-ui/src/simpos/views/report.js:25]: Why the relative import here?

**Review by netr0m — APPROVED** [review]

File: web/packages/plugins/simpos-ui/src/simpos/views/report.js (hunk 1 above)
Comment (soofstad) [reply web/packages/plugins/simpos-ui/src/simpos/views/report.js:25]: not intended. Pycharm refactor magic. Fixed!

File: web/packages/plugins/analysis-platform/src/components/Layout/Menu.css [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@import '~antd/dist/antd.css';
+@import '../../../../../../node_modules/antd/dist/antd.css';
```
Comment (netr0m) [inline web/packages/plugins/analysis-platform/src/components/Layout/Menu.css:1]: This one as well, then?

Comment (soofstad) [reply web/packages/plugins/analysis-platform/src/components/Layout/Menu.css:1]: ayyy. Fixed!

**Pull request merged**